src/auth/auth-config.ts

```diff
@@ -506,6 +507,28 @@ export interface MultiFactorConfig {
    * Currently only ‘phone’ is supported.
    */
   factorIds?: AuthFactorType[];
+  /**
+   * A list of multi-factor provider specific config. 
+   * New MFA providers (except phone) will indicate enablement/disablement through this field.
+   */
+  providerConfigs?: MultiFactorProviderConfig[];
+}
+
+export interface MultiFactorProviderConfig {
+  /* indicates whether this multi-factor provider is enabled/disabled. */
```

Comment:
I think this might need `/*** ....` style for the documentation to work correctly.